app/controllers/pages_controller.rb

```diff
@@ -21,13 +21,20 @@ def collect
   end
 
   def autocomplete
-    render json: Page.search(params[:query], {
-      fields: ["scientific_name^5"],
-      match: :word_start,
-      # limit: 10,
-      load: false,
-      misspellings: false
-    })
+    render json:
+      JSON.parse(Page.search((params[:query]), 
+        {
+          fields: ["name_string"], 
+          match: :word_start, 
+          load: false, 
```

Comment:
what is that mean?

Reply:
- It says [here](https://www.rubydoc.info/github/ankane/searchkick) that when "load" is set to false, this means that the results will be retrieved directly from ElasticSearch rather than from the database.
- It's also mentioned [here](https://github.com/ankane/searchkick#heres-how-to-make-it-work-with-rails) that it's best to set "load" to false for higher autocomplete performance.